### DISCUSSION TITLE
Theme UI Example

### DISCUSSION THREAD

**Discussion by guatedude2** [remix-run/remix#3636] [Proposals]
Hello, 
I'm interested in using Theme-UI with Remix. Is there an example to get SSR working with Theme-UI?

**Comment by kiliman** [comment]
Looks like Theme-UI is based on emotion. Here's a guide on getting emotion working with Remix. You should be able to go from there.

https://bensthoughts.dev/blog/remix/remix-emotion-styled

**Reply by guatedude2** [reply]
Unfortunately, that version of emotion doesn't quite work for Theme-UI. Mainly because the caching control is different for styled components thank it is for the JSX Pragma version. 

That's the main reason I decided to contribute this version.

**Reply by kiliman** [reply]
Ok, hopefully you'll be able to figure it out. I'm not sure anyone else has attempted it yet.

**Reply by guatedude2** [reply]
Yeah I got it working already. Took a while and it works. I posted a PR about it here https://github.com/remix-run/remix/pull/3637
